### ISSUE TITLE
Some referenced defenitions are not hosted in the OpenAPI specification file

### ISSUE THREAD

**Issue by mzvankovich** [ccouzens/vcloud-rest-openapi#4]
Some referenced defenitions are not hosted in the OpenAPI specification file. Thus the code (java) can not be generaed due to the lack of such types.

For examlpe the file [35.2.yml](https://github.com/ccouzens/vcloud-rest-openapi/blob/main/35.2.yml) with the following section:

```
ovf_VSSD_Type:
      title: ovf_VSSD_Type
      description: Wrapper for CIM\_VirtualSystemSettingData\_Type
      allOf:
        - $ref: '#/components/schemas/vssd_CIM_VirtualSystemSettingData_Type'
        - type: object
          additionalProperties: false
```

The **vssd_CIM_VirtualSystemSettingData_Type** is not presended in the file.

**Comment by ccouzens** [comment]
Hi,

Thank you for reporting. I should add some build -> client library tests to catch this sort of issue.

The parent type is `ovf_VSSD_Type`.

It looks like something must have changed at version 35.0 as `ovf` appears a lot more in the schemas at that point.

![image](https://user-images.githubusercontent.com/241046/173181097-fa0586e3-50fc-4b6d-8604-da64b5cc546e.png)

At the time I wrote this the `ovf` schemas did not follow the same patterns as the rest of the schema and were not important for my company's use case.

As such I excluded them
https://github.com/ccouzens/vcloud-rest-openapi/blob/main/transformer/src/schemas.rs#L20
And stubbed them out
https://github.com/ccouzens/vcloud-rest-openapi/blob/main/transformer/src/schema_tweaks/stub_ovf.rs

I need to look into if this is still the right decision for the newer schemas (I was mostly interested in 32.0) and why it has stopped working properly in later schemas.
